src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -318,8 +319,8 @@ public void validateArguments() {
      * the user and all tool defaults are disabled; {@code false} otherwise.
      */
     public boolean isDisabledFilter(final String filterName) {
-        return userDisabledReadFilterNames.contains(filterName)
-                || (disableToolDefaultReadFilters && !userEnabledReadFilterNames.contains(filterName));
+        return userArgs.getUserDisabledReadFilterNames().contains(filterName)
+                || (userArgs.disableToolDefaultReadFilters() && !userArgs.getUserEnabledReadFilterNames().contains(filterName));
```

Comment:
Maybe this logic will be better as an abstract method in the argument collection, called here.